lms/static/scripts/postmessage_json_rpc/client/client.js

```diff
@@ -0,0 +1,98 @@
+import { generateHexString } from './random';
```

Comment:
copied from the client and modified to use async, we should copy this back to the client or fold this into a shared util package.